docs/acsengine/kubernetes/AddMonitoringWorkspaceTag.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# login
+az login
+
+# set subscription of the acs-engine resource group
+az account set -s <subscriptionId>
+
+# check the log analytics workspace resource exists or not
+az resource show --ids "/subscriptions/72c8e8ca-dc16-47dc-b65c-6b5875eb600a/resourceGroups/gangams-acsengine-ws-rg/providers/Microsoft.OperationalInsights/workspaces/gangams-acsengine-workspace"
```

Comment:
Can you pls react the sub-id ?

Reply:
Also how to get the LA resourceID would be helpful.

Reply:
Good suggestion. Added details in the doc and also in the PS script.

Reply:
> Can you pls redact the sub-id ?

purged this sh file since we don't need.